pandera/pandera.py

```diff
@@ -208,6 +209,78 @@ def __call__(self, parent_schema, check_index, check_obj):
             return _vcheck(check_obj)
 
 
+class Hypothesis(Check):
+    """ Extends Check to cater for hypotheses validation"""
```

Comment:
consider changing this to something more descriptive of the class, e.g.
"Do a hypothesis test on a Column, potentially grouped by another column"

Reply:
@cosmicBboy have changed, please check